OneSignalSDK/unittest/src/test/java/com/test/onesignal/GenerateNotificationRunner.java

```diff
@@ -893,7 +884,7 @@ public void badgeCountShouldNotIncludeOldNotifications() {
       NotificationBundleProcessor_ProcessFromGCMIntentService(blankActivity, getBaseNotifBundle(), null);
 
       // Go forward 1 week
-      ShadowSystemClock.setCurrentTimeMillis(System.currentTimeMillis() + 604_801L * 1_000L);
+      advanceSystemTimeBy(604_801 * 1_000L);
```

Comment:
Lets include the units here. Maybe call it `advanceSystemTimeByMs`

Reply:
Only reason I wasnt adding units in the name is because the parameter is `long millis`

Reply:
Gotcha, I guess the param hint will end up showing in Android Studio with `millis` so it won't be an issue.